README.adoc

```diff
@@ -112,34 +112,41 @@ include::finish/src/main/java/io/openliberty/guides/consumingrest/model/Artist.j
 include::finish/src/main/java/io/openliberty/guides/consumingrest/model/Album.java[tags=!comment]
 ----
 
-=== Using Jackson
-
-In the case of using Jackson for JSON binding, a few restrictions are applicable to these objects,
-they:
+=== Using JSON-B
 
-* Cannot have a constructor unless it is empty.
-* Must contain the same number of instance members as there are keys in the target JSON.
+JSON-B is a new feature introduced with Java EE 8 and strengthens Java's support for JSON.
+This guide previously used Jackson for JSON binding.  JSON-B requires, by default, a
+public default no-arguments constructor for deserialization.
 
-While Jackson annotations are not always necessary, they are considered good practice when used:
+The JSON-B engine includes a set of default mapping rules which can be run without
+any customization annotations or custom configuration. There are instances where you may
+find it useful to deserialize a JSON message with only certain properties, specific
+property names, or have a class with a custom constructor. In these cases,
```

Comment:
When writing a list, try to make the items in the list parallel, such as all nouns, all verbs, etc. In this list, "have a class with a custom constructor" is the odd one out because all the other items in the list are nouns, but this last one starts with a verb, "have," instead. Could you change it by removing the "have" and use "classes with custom constructors"?